sdks/python/apache_beam/coders/proto2_coder_test_messages_pb2.py

```diff
@@ -17,16 +17,15 @@
 
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: sdks/java/core/src/main/proto/proto2_coder_test_messages.proto
-
-import sys
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
<!--new_thread; commit:8f4ed891ad8e0a0b3c7da031a7618c00545b64e5; resolved:0-->
Should we revert the changes in this generated file?

Also, which of these is really necessary?  Do we use division and print in this file?